src/com/projectkorra/projectkorra/chiblocking/RapidPunch.java

```diff
@@ -35,8 +35,10 @@ public RapidPunch(final Player sourceplayer, final Entity targetentity) {
 		this.cooldown = getConfig().getLong("Abilities.Chi.RapidPunch.Cooldown");
 		this.interval = getConfig().getLong("Abilities.Chi.RapidPunch.Interval");
 		this.target = targetentity;
-		this.bPlayer.addCooldown(this);
 		this.start();
+		if(!isRemoved()) {

```

Comment:
Missing a space between if and (

Reply:
Thanks, fixed in f10857212ba2492cfe0536748e6410b82316369b